pkgs/servers/freeradius/default.nix

```diff
@@ -1,28 +1,69 @@
-{ stdenv, fetchurl, autoreconfHook, talloc, openssl ? null }:
+{ stdenv, fetchurl, autoreconfHook, talloc
+, openssl
+, linkOpenssl? true
```

Comment:
I was following the same pattern that the openssh package follows. It's always a required dependency but linking against it can be turned off via linkOpenssl = false (which applies --without-openssl on the ./configure). 

I imagine it's not commonly used as the only way I see to toggle linkOpenssl in either openssh or freeradius is to use a package override.